mon-pix/app/styles/pages/_sign-form.scss

```
@@ -190,3 +186,17 @@
     font-size: 0.875rem;
   }
 }
+
+.pole-emploi {
+  $pole-emploi: #1B2E57;
+
+  color: $pix-neutral-0;
+  background-color: $pole-emploi;
+}
+
+.fwb {
+  $fwb: #EDEDED;
+
+  color: $pix-neutral-70;
+  background-color: $fwb;
+}
```

Pourquoi définir des variables SCSS qui ne servent qu'une fois ?

```suggestion
.pole-emploi {
  color: $pix-neutral-0;
  background-color: #1B2E57;
}

.fwb {
  color: $pix-neutral-70;
  background-color: #EDEDED;
}
```

---

Si ça te semble assez clair que ce sont des couleurs spécifiques aux idp, ça me va de les mettre directement. C'était histoire de bien marquer qu'elle ne font pas partie du design system

---

Oui la création des classes CSS spécifiques `pole-emploi` et `fwb` rend la chose très claire, donc pas besoin de variables SCSS.

Par contre ta réponse sur la recherche de clarté me fait penser que renommer ces classes CSS respectivement en `oidc-provider-pole-emploi` et `oidc-provider-fwb`.

---

done

---

Super 👍